screw_tool_m3:              # Name of screw fastening tool
  digital_in_port:       0  # The pin to which the pressure sensor is connected
  digital_out_port_suck: 0  # The pin to which the 'vac'  of pump is connected
  digital_out_port_blow: 1  # The pin to which the 'blow' of pump is connected
screw_tool_m4:
  digital_in_port:       1
  digital_out_port_suck: 2
  digital_out_port_blow: 3
suction_tool:
  digital_in_port:       2
  digital_out_port_suck: 4
  digital_out_port_blow: 5
base_fixture:
  digital_out_port_suck: 6
